src/Surfnet/StepupGateway/GatewayBundle/Resources/views/second_factor/choose_second_factor.html.twig

```diff
@@ -8,7 +8,12 @@
     <h2>{{ block('page_title') }}</h2>
     {{ form_start(form, {'attr': {'class': 'form-horizontal choose-second-factor-type'}}) }}
 
-    {{ form_errors(form) }}
+    {% set formErrors = form.vars.errors.form.getErrors(true) %}
+    {% if formErrors|length %}
+        {% for error in formErrors %}
```

Comment:
This is likely better of being a p-element.  Might mess up the styling a bit due to standard margin/padding on paragraphs though.

Reply:
Converted to P's for all templates.